support/special-thanks.md

```diff
@@ -90,3 +91,4 @@ They are passionate about their native language, and know that not everyone has
 * **Czech:** Esterze
 * **Russian:** Cloud
 * **Traditional Chinese:** AhMing, Lerp_Ant
+* **Spanish - Latam** v4ca
```

Comment:
There's a "Latin American Spanish" above, it's the same language, merge them and remember alphabetical order

Reply:
should be fixed